tests/test_experiments.py

```
@@ -231,9 +228,8 @@ def test_parameter_sweep_no_exact_gradients(self):
             Branch(function=lambda th: th**3),
         ]
         logits_model = LogitsModel(
-            logits_function=lambda th: jnp.array([0.0, 1.0]),
-            logits_derivative_function=lambda th: jnp.array(
-                [0.0, 0.0]
+            logits_function=lambda th: jnp.array(
```

Ruff (0.14.1)

Unused lambda argument: `th`

(ARG005)